### PR TITLE
Adds a space cleaner refiller to the janitor's closet. Other backpack water tank changes.

### PR DESCRIPTION
**Pull request by FlamingLily**
A.K.A. episode 2 of FlamingLily's Janitor Bonanza
Added a cleaner refiller to the wall in between the janitor lockers and biohazard lockers, moved the air alarm slightly.

Changed the name of the backpack water tank that lives in biohazard closets to reflect that it spawns with space cleaner.

### CODE REVIEW

**Comment by FlamingLily** [comment]
!review

**Review by JohnWildkins — APPROVED** [review]

File: code/game/objects/items/weapons/tanks/watertank.dm [hunk 1]
```diff
@@ -123,8 +123,8 @@
 
 //Janitor tank
 /obj/item/watertank/janitor
-	name = "backpack water tank"
-	desc = "A janitorial watertank backpack with nozzle to clean dirt and graffiti."
+	name = "backpack cleaner tank"
+	desc = "A janitorial cleanertank backpack with nozzle to clean dirt and graffiti."
```
Comment (SleepyGemmy) [inline code/game/objects/items/weapons/tanks/watertank.dm:127]:
```suggestion
	desc = "A janitorial cleaner tank that is worn on the back, with a nozzle to clean dirt and graffiti."
```

File: code/modules/reagents/reagent_dispenser.dm [hunk 2]
```diff
@@ -262,6 +262,13 @@
 	amount_per_transfer_from_this = 50
 	reagents_to_add = list(/decl/reagent/luminol = 1000)
 
+/obj/structure/reagent_dispensers/peppertank/spacecleaner
+	name = "space cleaner dispenser"
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_dispenser.dm:266]:
```suggestion
	name = "cleaner dispenser"
```
"space cleaner" is kind of a old, jokey name from when most things had "space" as a prefix. (e.g. "space milk", "space cola".) i personally don't think it should be added to new objects, if avoidable.

File: code/modules/reagents/reagent_dispenser.dm [hunk 3]
```diff
@@ -262,6 +262,13 @@
...
+	desc = "A dispenser to refill space cleaner bottles and backpack cleaner tanks."
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_dispenser.dm:267]:
```suggestion
	desc = "A wall-mounted dispenser filled with cleaner. Used to refill cleaner bottles and cleaner tanks."
```

**Review by SleepyGemmy — APPROVED** [review]
requested changes addressed.
code and mapping looks fine.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**